Doc/library/dis.rst

```diff
@@ -793,6 +798,49 @@ All of the following opcodes use their arguments.
    .. versionadded:: 3.6
 
 
+.. opcode:: BUILD_TUPLE_UNPACK (count)
+
+   Pops ``count`` iterables from the stack, joins them in a single tuple,
+   and pushes the result.  This bytecode is used for implementing iterable
+   unpacking in tuple displays ``(*x, *y, *z)``.
```

Comment:
Maybe add examples for other opcodes?

`[*x, *y, *z]`
`{*x, *y, *z}`
`{**x, **y, **z}`